package.json

```diff
@@ -1,15 +1,21 @@
 {
   "name": "vue-a11y-dialog",
   "description": "Vue.js component wrapping for a11y-dialog",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "homepage": "https://github.com/morkro/vue-a11y-dialog",
   "license": "MIT",
   "author": "Moritz Kröger (https://moritz.berlin)",
   "private": false,
-  "main": "dist/vue-a11y-dialog.js",
-  "files": [
-    "dist/vue-a11y-dialog.js"
-  ],
+  "files": ["dist"],
+  "main": "dist/vue-a11y-dialog.umd.js",
+  "module": "dist/vue-a11y-dialog.esm.js",
+  "browser": "dist/vue-a11y-dialog.umd.js",
+  "exports": {
+    ".": {
+      "import": "./dist/vue-a11y-dialog.esm.js",
+      "require": "./dist/vue-a11y-dialog.umd.js"
```

Comment:
Should we provide the cjs one too? I believe that's still getting generated to dist:

![image](https://user-images.githubusercontent.com/142403/150976594-91c28668-c8bc-4413-9798-ee8d1308bed6.png)

I'm not sure if many folks will actually use this in CJS but since it's easy enough to generate and it's there I think we should export it here.

Reply:
Alternatively, we could just remove it from the vite.config.js build entirely

Reply:
I'll remove it